src/storage/page_buffer.c

```diff
@@ -887,6 +887,10 @@ struct pgbuf_fix_perf
   UINT64 fix_wait_time;
 };
 
+#define LSA_IS_INIT_NONTEMP(lsa_ptr) LSA_ISNULL(lsa_ptr)
```

Comment:
There are similar definitions in disk_manager.c.
What about putting them to log_lsa.hpp ?

Reply:
I purposely avoided to put them to log_lsa.hpp since it is related to storage module.

I removed the macro from disk_manager.c and added `const log_lsa PGBUF_TEMP_LSA` to page_buffer.h.

Side changes:

  - remove unused pgbuf_set_lsa_as_permanent
  - move and rename fileio_init_lsa_of_temp_page as pgbuf_init_temp_page_lsa.